release-notes/release-notes59.md

```diff
@@ -45,6 +45,78 @@ For information on:
+
+`tablecellvalign`
+: Allows the user to set the vertical alignment for the content of selected cells.
+
+`tablecellborderwidth`
+: Allows the user to set the border width on selected table cells. The available widths are set with the [`table_border_widths` option]({{site.baseurl}}/plugins/opensource/table/#table_border_widths).
+
+`tablecellborderstyle`
+: Allows the user to set the border style on selected table cells. The available styles are set with the [`table_border_styles` option]({{site.baseurl}}/plugins/opensource/table/#table_border_styles).
+
+`tablecaption`
+: Toggles the caption on the selected table.
+
+`tablecellbackgroundcolor`
+: Allows the user to change the background color of selected cells. The available colors are set by the [`table_cell_background_color_map` option]({{site.baseurl}}/plugins/opensource/table/#table_cell_background_color_map) or [`color_map` option]({{site.baseurl}}/plugins/opensource/table/#color_map).
+
+`tablecellbordercolor`
+: Allows the user to change the border color of selected cells. The available colors are set by the [`table_cell_border_color_map` option]({{site.baseurl}}/plugins/opensource/table/#table_cell_border_color_map) or [`color_map` option]({{site.baseurl}}/plugins/opensource/table/#color_map).
+
+`tablerowheader`
+: Allows the user to toggle selected rows into headers.
+
+`tablecolheader`
+: Allows the user to toggle selected columns into headers.
+
+For information on these buttons, see: [Available toolbar buttons]({{site.baseurl}}/advanced/available-toolbar-buttons/#tableplugin)
+For information on these menu items, see: [Available menu items]({{site.baseurl}}/advanced/available-menu-items/#tableplugin)
+
+#### New table options
+
+{{site.productname}} 5.9 adds new options for the Table plugin to improve the user experience when working with tables. These new options are optional, and allows further customization of user experience.
+
+`table_cell_background_color_map`
+: Allows setting a specific set of colors to be used by the `tablecellbackgroundcolor` toolbar button and menu item, overriding the defaults and the colors provided by the [`color_map` option]({{site.baseurl}}/plugins/opensource/table/#color_map). For information on this option, see: [Table options - table_cell_background_color_map]({{site.baseurl}}/plugins/opensource/table/#table_cell_background_color_map)
+
+`table_cell_border_color_map`
+: Allows setting a specific set of colors to be used by the `tablecellbordercolor` toolbar button and menu item, overriding the defaults and the colors provided by the [`color_map` option]({{site.baseurl}}/plugins/opensource/table/#color_map). For information on this option, see: [Table options - table_cell_border_color_map]({{site.baseurl}}/plugins/opensource/table/#table_cell_border_color_map)
```

Comment:
for added clarity

Allows setting a specific set of border colors

Reply:
```suggestion
: Allows setting a specific set of border colors to be used by the `tablecellbordercolor` toolbar button and menu item, overriding the defaults and the colors provided by the [`color_map` option]({{site.baseurl}}/plugins/opensource/table/#color_map). For information on this option, see: [Table options - table_cell_border_color_map]({{site.baseurl}}/plugins/opensource/table/#table_cell_border_color_map)
```